doc/base/classes.xml

```diff
@@ -12365,6 +12365,18 @@
 				Return the list of keys in the [Dictionary].
 			</description>
 		</method>
+		<method name="parse_json">
+			<return type="int">
+			</return>
+			<argument index="0" name="json" type="String">
+			</argument>
+			<description>
+				Parse json string into the dictionary. Return OK when successed or the error code when failed.
+				
```

Comment:
Just a style comment: the in-editor help and the online docs consider each line break as a new paragraph, so the current text will end up as:
```
Parse json string into the dictionary. Return OK when successed or the error code when failed.



Be aware that the JSON specification does not define integer or float types, but only a number type.

Therefore, parsing a JSON text into a dictionary will convert every numerical values to [float] types.
```

You might want to change the XML to:
```
Parse json string into the dictionary. Return OK when successed or the error code when failed.
Be aware that the JSON specification does not define integer or float types, but only a number type. Therefore, parsing a JSON text into a dictionary will convert every numerical values to [float] types.
```